druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/dynamic/RocksDBDynamicLookupExtractor.java

```diff
@@ -35,6 +43,50 @@ public boolean canIterate() {
 
     @Override
     public Iterable<Map.Entry<String, String>> iterable() {
-        return staticMap.entrySet();
+        Map<String, String> tempMap = new java.util.HashMap<>();
+
+        try {
+            final RocksDB db = rocksDBManager.getDB(extractionNamespace.getNamespace());
+            RocksIterator it = db.newIterator();
+            it.seekToFirst();
+            while (it.isValid()) {
+                Optional<DynamicLookupSchema> dynamicLookupSchemaOption = schemaManager.getSchema(extractionNamespace);
+                if(!dynamicLookupSchemaOption.isPresent()) {
+                    return tempMap.entrySet();
+                }
+
+                byte[] cacheByteValue = db.get(it.key());
+                if (cacheByteValue == null) {
```

Comment:
if one value is null, it should continue looking for next values.

Reply:
Fixed